pom.xml

```diff
@@ -9,6 +9,8 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <kotlin.version>1.3.41</kotlin.version>
+        <junit.jupiter.version>5.6.1</junit.jupiter.version>
```

Comment:
Why create a property if the version is used only once?

Reply:
okay I will remove these properties